app/views/state_file/questions/nc_review/edit.html.erb

```diff
@@ -34,5 +34,13 @@
     </div>
   </div>
 
+  <div id="state-credits" class="white-group">
+    <div class="spacing-below-5">
+      <p class="text--bold spacing-below-5"><%=t(".state_credit") %></p>
+      <p><%=current_intake.tribal_member_yes? ? number_to_currency(current_intake.tribal_wages_amount): t("general.negative") %></p>
```

Comment:
can we do `number_to_currency(current_intake.tribal_wages_amount, precision: 2)` here so we show the cents value if they do enter it?